frontend/lib/loc/letter-request.tsx

```diff
@@ -26,26 +28,28 @@ export const SendConfirmModal = withAppContext(
 
     return (
       <Modal
-        title="Ready to go"
+        title={li18n._(t`Ready to go`)}
         withHeading
         onCloseGoTo={BackOrUpOneDirLevel}
         render={(ctx) => (
           <>
             <p>
-              JustFix.nyc will send this letter via USPS Certified Mail
-              <sup>&reg;</sup> <strong>within 1-2 business days</strong> to your
-              landlord:
+              <Trans>
+                JustFix.nyc will send this letter via USPS Certified Mail
+                <sup>&reg;</sup> <strong>within 1-2 business days</strong> to
+                your landlord:
+              </Trans>
             </p>
             <address className="has-text-centered">
-              {landlord.name || "UNKNOWN LANDLORD"}
+              {landlord.name || li18n._(t`UNKNOWN LANDLORD`)}
               <br />
-              {landlord.address || "UNKNOWN ADDRESS"}
+              {landlord.address || li18n._(t`UNKNOWN ADDRESS`)}
```

Comment:
Oops don't worry about internationalizing these--this text is actually never supposed to show up in real-world situations, so we shouldn't burden translators with having to translate it.